domains/authorization/Authorization.API/API/Models/Organization.cs

```diff
@@ -48,6 +52,14 @@ OrganizationName organizationName
         );
     }
 
+    public void AcceptServiceProviderTerms()
+    {
+        ServiceProviderTermsAccepted = true;
+        ServiceProviderTermsAcceptanceDate = UnixTimestamp.Now();
+    }
+
+    public void InvalidateServiceProviderTerms() => ServiceProviderTermsAccepted = false;
```

Comment:
InvalidateServiceProviderTerms -> RevokeServiceProviderTerms? (nit pick, not important)
Should we update: ServiceProviderTermsAcceptanceDate aswell?

Reply:
Godt spørgsmål. Det er lavet så det følger det gamle terms. Og dengang var tanken at hvis man invalidater terms, så kunne det være fint at se, hvornår organization har accepteret terms sidste gang. Lad os bare følge det vi har gjort tidligere tænker jeg, for at undgå forvirring.

Mht. navngivning har du nok ret. Har renamed den: 

https://github.com/Energinet-DataHub/energy-origin/pull/1605/commits/c1f75edf5897b981542251ef63c8fa13e80fe1c9